api/src/main/java/ca/bc/gov/educ/api/distribution/process/BaseProcess.java

```diff
@@ -169,12 +170,12 @@ protected void processSchoolsForLabels(List<School> schools, Psi psi) {
         schools.add(school);
     }
 
-    protected void processSchoolsForLabels(String recipient, List<School> schools, String mincode, ExceptionMessage exception) {
-        School existSchool = schools.stream().filter(s -> mincode.equalsIgnoreCase(s.getMincode())).findAny().orElse(null);
+    protected void processSchoolsForLabels(String recipient, List<ca.bc.gov.educ.api.distribution.model.dto.School> schools, String mincode, ExceptionMessage exception) {
+        ca.bc.gov.educ.api.distribution.model.dto.School existSchool = schools.stream().filter(s -> mincode.equalsIgnoreCase(s.getMincode())).findAny().orElse(null);
         if (existSchool != null) return;
-        TraxSchool traxSchool = schoolService.getTraxSchool(mincode, exception);
+        ca.bc.gov.educ.api.distribution.model.dto.v2.School traxSchool = schoolService.getSchool(mincode, exception);
         if (traxSchool != null) {
-            School school = new School();
+            ca.bc.gov.educ.api.distribution.model.dto.School school = new ca.bc.gov.educ.api.distribution.model.dto.School();
```

Comment:
Should these all be v2 schools?

Reply:
This is right as School DTO here is for Report Data being used by Grad-Report-Api.